backend/vkt/src/test/java/fi/oph/vkt/service/ClerkExamEventServiceTest.java

```diff
@@ -280,12 +281,12 @@ public void testCreateExamEvent() {
 
     assertEquals(examEvent.getId(), examEventDTO.id());
     assertEquals(examEvent.getVersion(), examEventDTO.version());
```

Comment:
Näillä riveillä voi myös parametrien järjestyksen vaihtaa mutta loppupeleissä aikast lailla sama kummin päin ovat

Reply:
Niin, eihän tuolla oikeasti väliä ole, mutta id ja version on mielestäni tuossa nyt oikein päin.